data_steward/curation_logging/slack_logging_handler.py

```diff
@@ -0,0 +1,44 @@
+# Python imports
+import os
+import logging
+
+# Project imports
+from utils.slack_alerts import post_message, check_channel_and_token, SlackConfigurationError, SLACK_CHANNEL, SLACK_TOKEN
+
+
+class SlackLoggingHandler(logging.Handler):
+    """
+     Logging handler to send messages to a Slack Channel.
+    """
+
+    def __init__(self):
+        super().__init__(level=logging.WARNING)
+
+    def emit(self, record):
+
+        try:
+            if not _is_raised_from_itself(record):
```

Comment:
Just a comment to explain why we add this check is here. 

```suggestion
            # this is added for preventing the infinite loop from happening
            if not _is_raised_from_itself(record):
```

Reply:
Yep, because I will definitely forget :p